x/group/simulation/operations.go

```diff
@@ -0,0 +1,748 @@
+			fees,
+			helpers.DefaultGenTxGas,
+			chainID,
+			[]uint64{account.GetAccountNumber()},
+			[]uint64{account.GetSequence()},
+			acc1.PrivKey,
+		)
+		if err != nil {
+			return simtypes.NoOpMsg(group.ModuleName, group.TypeMsgUpdateGroupAccountDecisionPolicy, "unable to generate mock tx"), nil, err
+		}
+
+		_, _, err = app.Deliver(txGen.TxEncoder(), tx)
+		if err != nil {
+			return simtypes.NoOpMsg(group.ModuleName, msg.Type(), "unable to deliver tx"), nil, err
+		}
+
+		return simtypes.NewOperationMsg(msg, true, ""), nil, err
+	}
+}
+
+// SimulateMsgUpdateGroupAccountComment generates a MsgUpdateGroupAccountMetadataRequest with random values
+func SimulateMsgUpdateGroupAccountComment(ak exported.AccountKeeper,
+	bk exported.BankKeeper, queryClient group.QueryClient) simtypes.Operation {
+	return func(
+		r *rand.Rand, app *baseapp.BaseApp, sdkCtx sdk.Context, accounts []simtypes.Account, chainID string) (simtypes.OperationMsg, []simtypes.FutureOperation, error) {
+		acc1 := accounts[0]
+
+		account := ak.GetAccount(sdkCtx, acc1.Address)
+
+		spendableCoins := bk.SpendableCoins(sdkCtx, account.GetAddress())
+		fees, err := simtypes.RandomFees(r, sdkCtx, spendableCoins)
+		if err != nil {
+			return simtypes.NoOpMsg(group.ModuleName, group.TypeMsgUpdateGroupAccountComment, "fee error"), nil, err
+		}
+
+		ctx := regentypes.Context{Context: sdkCtx}
+		groupAdmin, _, op, err := getGroupDetails(sdkCtx, queryClient, acc1)
+		if err != nil {
+			return op, nil, err
+		}
+		if groupAdmin == "" {
+			return op, nil, nil
+		}
+
+		result, err := queryClient.GroupAccountsByAdmin(ctx, &group.QueryGroupAccountsByAdminRequest{Admin: groupAdmin})
+		if err != nil {
+			return simtypes.NoOpMsg(group.ModuleName, group.TypeMsgUpdateGroupAccountComment, "fail to query group info"), nil, err
+		}
+
+		groupAccounts := result.GetGroupAccounts()
+		if len(groupAccounts) == 0 {
+			return simtypes.NoOpMsg(group.ModuleName, group.TypeMsgUpdateGroupAccountComment, "no group account found"), nil, nil
+		}
+
+		msg := group.MsgUpdateGroupAccountMetadataRequest{
+			Admin:        groupAccounts[0].Admin,
+			GroupAccount: groupAccounts[0].GroupAccount,
+			Metadata:     []byte(simtypes.RandStringOfLength(r, 10)),
+		}
+
+		txGen := simappparams.MakeTestEncodingConfig().TxConfig
+		tx, err := helpers.GenTx(
+			txGen,
+			[]sdk.Msg{&msg},
+			fees,
+			helpers.DefaultGenTxGas,
+			chainID,
+			[]uint64{account.GetAccountNumber()},
+			[]uint64{account.GetSequence()},
+			acc1.PrivKey,
+		)
+		if err != nil {
+			return simtypes.NoOpMsg(group.ModuleName, group.TypeMsgUpdateGroupAccountComment, "unable to generate mock tx"), nil, err
+		}
+
+		_, _, err = app.Deliver(txGen.TxEncoder(), tx)
+		if err != nil {
+			return simtypes.NoOpMsg(group.ModuleName, msg.Type(), "unable to deliver tx"), nil, err
+		}
+
+		return simtypes.NewOperationMsg(&msg, true, ""), nil, err
+	}
+}
+
+// TODO: Add WeightedOperations for MsgVote & MsgExec. Waiting on genesis PR
```

Comment:
why are you waiting on genesis PR? If it's just about adding `proposal_id` to a `Proposal`, you can go ahead directly here.